x-pack/plugins/security_solution/cypress/objects/case.ts

```diff
@@ -44,3 +44,189 @@ export const serviceNowConnector: Connector = {
 };
 
 export const TIMELINE_CASE_ID = '68248e00-f689-11ea-9ab2-59238b522856';
+export const ids = {
```

Comment:
nit -> I think is better to rename it to `connectorIds` or `ids: { connectors: { ... } }`